tests/unit/models/test_settings.py

```diff
@@ -0,0 +1,51 @@
+import os
+import tempfile
+from pathlib import Path
+
+from pytest import fixture
+
+from runner_manager.models.settings import (
+    ConfigFile,
+    Settings,
+    yaml_config_settings_source,
+)
+
+
+@fixture
+def test_settings_default_values():
+    settings = Settings()
+    assert settings.name == "runner-manager"
+    assert settings.redis_om_url == os.getenv("REDIS_OM_URL")
```

Comment:
If we don't have those var set it won't work.

And I don't think we should force user to set it, the way it was before with the hardcoded value was fine I believe.

Reply:
tested and actually the value is set to None if the value is not set so the test is good.